lib/puppet/module.rb

```diff
@@ -351,6 +361,25 @@ def ==(other)
     self.environment == other.environment
   end
 
+  def initialize_i18n
+    locales_path = File.absolute_path('locales', path)
+    config_path = File.absolute_path('config.yaml', locales_path)
+    module_name = @forge_name.gsub("/","-") if @forge_name
+
+    nil if initialized_i18n? module_name
+
+    begin
+      GettextSetup.initialize(locales_path) unless !File.file? config_path
+      Puppet.debug _("%{value0,} initialized for i18n: %{value1,}") % { value0: module_name, value1: GettextSetup.translation_repositories[module_name]}
```

Comment:
Also for future reference, please try to replace the `value0` etc. with meaningful names. This will make it easier for the translator. Also as written, the commas will cause a mismatch of keys and this will explode.